src/frontend/admin/src/services/api/models/Organization.ts

```diff
@@ -19,6 +19,20 @@ export type Organization = {
   contact_phone?: string;
   contact_email?: string;
   dpo_email?: string;
+  addresses?: OrganizationAddress[];
+};
+
+export type OrganizationAddress = {
+  id: string;
+  title: string;
+  address: string;
+  postcode: string;
+  city: string;
+  country: string;
```

Comment:
Proposal: It would be great to have an enum for this, WDYT?

Reply:
Currently, it is an option query which gives the whole list of possibilities, to avoid maintaining back and grund